force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -321,18 +414,27 @@ public with sharing class FlowControlLoopsCollections {
      * @return The salary of the employee, or null if the employee is not found in the Map.
      */
     public static Integer getSalary(String employeeName) {
+        Map<String, Integer> employeeSalaryMap = new Map<String, Integer>();
+        
         // Initialize a Map
 
         // Add key-value pairs to the Map
         // add John Doe => 50000
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
-
+        employeeSalaryMap.put('John Doe', 50000);
+        employeeSalaryMap.put('Jane Smith', 60000);
+        employeeSalaryMap.put('Sam Brown', 55000);
+        employeeSalaryMap.put('Alice Johnson', 65000);
         // Get the salary of the employee
-
-        // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        for(Integer i = 0; i < employeeSalaryMap.size(); i++){
```

Comment:
You do not need to loop here. The containKey already looks for the key in the map.